app/js/first/directives/title_directive.js

```diff
@@ -0,0 +1,13 @@
+module.exports = function(app) {
+  app.directive('firstImageDirective', function() {
```

Comment:
For consistency you should name your directive the same as the file name.  That way if someone is tracing your code looking for 'firstImageDirective' they will know where to look.  Think of yourself six months from now looking back at your code and trying to remember where you put things.